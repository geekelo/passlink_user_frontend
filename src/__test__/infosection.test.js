import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter, useLocation } from 'react-router-dom';
import InfoSection from '../homePage/javascripts/infosection';

// Mock useLocation to return a pathname for testing
jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useLocation: jest.fn(),
}));

test('renders InfoSection component correctly for /about route', () => {
  useLocation.mockReturnValue({ pathname: '/about' });
  const { asFragment } = render(
    <BrowserRouter>
      <InfoSection />
    </BrowserRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});

test('renders InfoSection component correctly for /contact route', () => {
  useLocation.mockReturnValue({ pathname: '/contact' });
  const { asFragment } = render(
    <BrowserRouter>
      <InfoSection />
    </BrowserRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
