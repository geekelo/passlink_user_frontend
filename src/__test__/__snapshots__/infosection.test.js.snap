// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders InfoSection component correctly for /about route 1`] = `
<DocumentFragment>
  <div
    class="info-section"
  >
    <div
      class="infoNav-section"
    >
      <div
        class="back-button"
      >
        <a
          aria-current="page"
          class="links infonav-icons active"
          href="/"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-house "
            data-icon="house"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 576 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M575.8 255.5c0 18-15 32.1-32 32.1h-32l.7 160.2c0 2.7-.2 5.4-.5 8.1V472c0 22.1-17.9 40-40 40H456c-1.1 0-2.2 0-3.3-.1c-1.4 .1-2.8 .1-4.2 .1H416 392c-22.1 0-40-17.9-40-40V448 384c0-17.7-14.3-32-32-32H256c-17.7 0-32 14.3-32 32v64 24c0 22.1-17.9 40-40 40H160 128.1c-1.5 0-3-.1-4.5-.2c-1.2 .1-2.4 .2-3.6 .2H104c-22.1 0-40-17.9-40-40V360c0-.9 0-1.9 .1-2.8V287.6H32c-18 0-32-14-32-32.1c0-9 3-17 10-24L266.4 8c7-7 15-8 22-8s15 2 21 7L564.8 231.5c8 7 12 15 11 24z"
              fill="currentColor"
            />
          </svg>
        </a>
      </div>
      <div
        class="back-button"
      >
        <div>
          <button
            aria-label="nav-open"
            class="infonav-btn opendropdown-icon"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-bars infonav-icons"
              data-icon="bars"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 96C0 78.3 14.3 64 32 64H416c17.7 0 32 14.3 32 32s-14.3 32-32 32H32C14.3 128 0 113.7 0 96zM0 256c0-17.7 14.3-32 32-32H416c17.7 0 32 14.3 32 32s-14.3 32-32 32H32c-17.7 0-32-14.3-32-32zM448 416c0 17.7-14.3 32-32 32H32c-17.7 0-32-14.3-32-32s14.3-32 32-32H416c17.7 0 32 14.3 32 32z"
                fill="currentColor"
              />
            </svg>
          </button>
          <button
            aria-label="nav-close"
            class="infonav-btn closedropdown-icon hide"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-xmark infonav-icons"
              data-icon="xmark"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 384 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M342.6 150.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L192 210.7 86.6 105.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L146.7 256 41.4 361.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L192 301.3 297.4 406.6c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L237.3 256 342.6 150.6z"
                fill="currentColor"
              />
            </svg>
          </button>
        </div>
        <div
          class="dropdown-section hide"
        >
          <a
            class="infonav-links links"
            href="/about"
          >
            About
          </a>
          <a
            class="infonav-links links"
            href="/contact"
          >
            Contact
          </a>
          <a
            class="infonav-links links"
            href="/features"
          >
            Features
          </a>
          <a
            class="infonav-links links"
            href="/privacy-policy"
          >
            Privacy Policy
          </a>
          <a
            class="infonav-links links"
            href="/faqs"
          >
            FAQs | Help
          </a>
        </div>
      </div>
    </div>
    <h2
      class="content-title"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-circle-info "
        data-icon="circle-info"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM216 336h24V272H216c-13.3 0-24-10.7-24-24s10.7-24 24-24h48c13.3 0 24 10.7 24 24v88h8c13.3 0 24 10.7 24 24s-10.7 24-24 24H216c-13.3 0-24-10.7-24-24s10.7-24 24-24zm40-208a32 32 0 1 1 0 64 32 32 0 1 1 0-64z"
          fill="currentColor"
        />
      </svg>
       About
    </h2>
    <div
      class="section"
    >
      <h3
        class="section-title"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-bullseye "
          data-icon="bullseye"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M448 256A192 192 0 1 0 64 256a192 192 0 1 0 384 0zM0 256a256 256 0 1 1 512 0A256 256 0 1 1 0 256zm256 80a80 80 0 1 0 0-160 80 80 0 1 0 0 160zm0-224a144 144 0 1 1 0 288 144 144 0 1 1 0-288zM224 256a32 32 0 1 1 64 0 32 32 0 1 1 -64 0z"
            fill="currentColor"
          />
        </svg>
         Our Mission
      </h3>
      <p
        class="section-content"
      >
        At PassLink, our mission is to revolutionize the event management industry by providing innovative solutions that streamline the registration process and enhance the overall event experience for guests and organizers. We are committed to delivering cutting-edge technology that simplifies event check-in procedures, maximizes efficiency, and empowers organizations to make data-driven decisions for success.
      </p>
      <div
        class="copybtnSection"
      >
        <button
          class="copybtn"
          type="submit"
        >
          <span>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-copy "
              data-icon="copy"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M208 0H332.1c12.7 0 24.9 5.1 33.9 14.1l67.9 67.9c9 9 14.1 21.2 14.1 33.9V336c0 26.5-21.5 48-48 48H208c-26.5 0-48-21.5-48-48V48c0-26.5 21.5-48 48-48zM48 128h80v64H64V448H256V416h64v48c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V176c0-26.5 21.5-48 48-48z"
                fill="currentColor"
              />
            </svg>
              Copy
          </span>
        </button>
      </div>
      <p />
    </div>
    <div
      class="section"
    >
      <h3
        class="section-title"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-eye "
          data-icon="eye"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M288 32c-80.8 0-145.5 36.8-192.6 80.6C48.6 156 17.3 208 2.5 243.7c-3.3 7.9-3.3 16.7 0 24.6C17.3 304 48.6 356 95.4 399.4C142.5 443.2 207.2 480 288 480s145.5-36.8 192.6-80.6c46.8-43.5 78.1-95.4 93-131.1c3.3-7.9 3.3-16.7 0-24.6c-14.9-35.7-46.2-87.7-93-131.1C433.5 68.8 368.8 32 288 32zM144 256a144 144 0 1 1 288 0 144 144 0 1 1 -288 0zm144-64c0 35.3-28.7 64-64 64c-7.1 0-13.9-1.2-20.3-3.3c-5.5-1.8-11.9 1.6-11.7 7.4c.3 6.9 1.3 13.8 3.2 20.7c13.7 51.2 66.4 81.6 117.6 67.9s81.6-66.4 67.9-117.6c-11.1-41.5-47.8-69.4-88.6-71.1c-5.8-.2-9.2 6.1-7.4 11.7c2.1 6.4 3.3 13.2 3.3 20.3z"
            fill="currentColor"
          />
        </svg>
         Our Vision
      </h3>
      <p
        class="section-content"
      >
        We envision a future where event management is seamless, efficient, and data-driven. By leveraging the power of technology, we aim to transform the way events are organized and attended, making the process more convenient, engaging, and insightful for all stakeholders. Our vision is to set new standards of excellence in event management and create a world where every event is a success.
      </p>
      <div
        class="copybtnSection"
      >
        <button
          class="copybtn"
          type="submit"
        >
          <span>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-copy "
              data-icon="copy"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M208 0H332.1c12.7 0 24.9 5.1 33.9 14.1l67.9 67.9c9 9 14.1 21.2 14.1 33.9V336c0 26.5-21.5 48-48 48H208c-26.5 0-48-21.5-48-48V48c0-26.5 21.5-48 48-48zM48 128h80v64H64V448H256V416h64v48c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V176c0-26.5 21.5-48 48-48z"
                fill="currentColor"
              />
            </svg>
              Copy
          </span>
        </button>
      </div>
      <p />
    </div>
    <div
      class="section"
    >
      <h3
        class="section-title"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-clipboard-list "
          data-icon="clipboard-list"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 384 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M192 0c-41.8 0-77.4 26.7-90.5 64H64C28.7 64 0 92.7 0 128V448c0 35.3 28.7 64 64 64H320c35.3 0 64-28.7 64-64V128c0-35.3-28.7-64-64-64H282.5C269.4 26.7 233.8 0 192 0zm0 64a32 32 0 1 1 0 64 32 32 0 1 1 0-64zM72 272a24 24 0 1 1 48 0 24 24 0 1 1 -48 0zm104-16H304c8.8 0 16 7.2 16 16s-7.2 16-16 16H176c-8.8 0-16-7.2-16-16s7.2-16 16-16zM72 368a24 24 0 1 1 48 0 24 24 0 1 1 -48 0zm88 0c0-8.8 7.2-16 16-16H304c8.8 0 16 7.2 16 16s-7.2 16-16 16H176c-8.8 0-16-7.2-16-16z"
            fill="currentColor"
          />
        </svg>
         Our Values
      </h3>
      <p
        class="section-content"
      >
        PassLink is built on a foundation of integrity, innovation, and excellence. We are committed to upholding the highest standards of ethics and professionalism in everything we do. We embrace creativity
      </p>
      <div
        class="copybtnSection"
      >
        <button
          class="copybtn"
          type="submit"
        >
          <span>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-copy "
              data-icon="copy"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M208 0H332.1c12.7 0 24.9 5.1 33.9 14.1l67.9 67.9c9 9 14.1 21.2 14.1 33.9V336c0 26.5-21.5 48-48 48H208c-26.5 0-48-21.5-48-48V48c0-26.5 21.5-48 48-48zM48 128h80v64H64V448H256V416h64v48c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V176c0-26.5 21.5-48 48-48z"
                fill="currentColor"
              />
            </svg>
              Copy
          </span>
        </button>
      </div>
      <p />
    </div>
    <div
      class="section"
    >
      <h3
        class="section-title"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-gears "
          data-icon="gears"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 640 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M308.5 135.3c7.1-6.3 9.9-16.2 6.2-25c-2.3-5.3-4.8-10.5-7.6-15.5L304 89.4c-3-5-6.3-9.9-9.8-14.6c-5.7-7.6-15.7-10.1-24.7-7.1l-28.2 9.3c-10.7-8.8-23-16-36.2-20.9L199 27.1c-1.9-9.3-9.1-16.7-18.5-17.8C173.9 8.4 167.2 8 160.4 8h-.7c-6.8 0-13.5 .4-20.1 1.2c-9.4 1.1-16.6 8.6-18.5 17.8L115 56.1c-13.3 5-25.5 12.1-36.2 20.9L50.5 67.8c-9-3-19-.5-24.7 7.1c-3.5 4.7-6.8 9.6-9.9 14.6l-3 5.3c-2.8 5-5.3 10.2-7.6 15.6c-3.7 8.7-.9 18.6 6.2 25l22.2 19.8C32.6 161.9 32 168.9 32 176s.6 14.1 1.7 20.9L11.5 216.7c-7.1 6.3-9.9 16.2-6.2 25c2.3 5.3 4.8 10.5 7.6 15.6l3 5.2c3 5.1 6.3 9.9 9.9 14.6c5.7 7.6 15.7 10.1 24.7 7.1l28.2-9.3c10.7 8.8 23 16 36.2 20.9l6.1 29.1c1.9 9.3 9.1 16.7 18.5 17.8c6.7 .8 13.5 1.2 20.4 1.2s13.7-.4 20.4-1.2c9.4-1.1 16.6-8.6 18.5-17.8l6.1-29.1c13.3-5 25.5-12.1 36.2-20.9l28.2 9.3c9 3 19 .5 24.7-7.1c3.5-4.7 6.8-9.5 9.8-14.6l3.1-5.4c2.8-5 5.3-10.2 7.6-15.5c3.7-8.7 .9-18.6-6.2-25l-22.2-19.8c1.1-6.8 1.7-13.8 1.7-20.9s-.6-14.1-1.7-20.9l22.2-19.8zM112 176a48 48 0 1 1 96 0 48 48 0 1 1 -96 0zM504.7 500.5c6.3 7.1 16.2 9.9 25 6.2c5.3-2.3 10.5-4.8 15.5-7.6l5.4-3.1c5-3 9.9-6.3 14.6-9.8c7.6-5.7 10.1-15.7 7.1-24.7l-9.3-28.2c8.8-10.7 16-23 20.9-36.2l29.1-6.1c9.3-1.9 16.7-9.1 17.8-18.5c.8-6.7 1.2-13.5 1.2-20.4s-.4-13.7-1.2-20.4c-1.1-9.4-8.6-16.6-17.8-18.5L583.9 307c-5-13.3-12.1-25.5-20.9-36.2l9.3-28.2c3-9 .5-19-7.1-24.7c-4.7-3.5-9.6-6.8-14.6-9.9l-5.3-3c-5-2.8-10.2-5.3-15.6-7.6c-8.7-3.7-18.6-.9-25 6.2l-19.8 22.2c-6.8-1.1-13.8-1.7-20.9-1.7s-14.1 .6-20.9 1.7l-19.8-22.2c-6.3-7.1-16.2-9.9-25-6.2c-5.3 2.3-10.5 4.8-15.6 7.6l-5.2 3c-5.1 3-9.9 6.3-14.6 9.9c-7.6 5.7-10.1 15.7-7.1 24.7l9.3 28.2c-8.8 10.7-16 23-20.9 36.2L315.1 313c-9.3 1.9-16.7 9.1-17.8 18.5c-.8 6.7-1.2 13.5-1.2 20.4s.4 13.7 1.2 20.4c1.1 9.4 8.6 16.6 17.8 18.5l29.1 6.1c5 13.3 12.1 25.5 20.9 36.2l-9.3 28.2c-3 9-.5 19 7.1 24.7c4.7 3.5 9.5 6.8 14.6 9.8l5.4 3.1c5 2.8 10.2 5.3 15.5 7.6c8.7 3.7 18.6 .9 25-6.2l19.8-22.2c6.8 1.1 13.8 1.7 20.9 1.7s14.1-.6 20.9-1.7l19.8 22.2zM464 304a48 48 0 1 1 0 96 48 48 0 1 1 0-96z"
            fill="currentColor"
          />
        </svg>
         Description
      </h3>
      <p
        class="section-content"
      >
        PassLink is a leading event management platform that offers a comprehensive suite of tools and services to help organizations plan, promote, and execute successful events. Our platform is designed to simplify the event planning process, enhance the attendee experience, and provide valuable insights to event organizers. With PassLink, you can create custom event websites, manage registrations, sell tickets, track attendance, and much more. Whether you are hosting a small gathering or a large conference, PassLink has the tools you need to make your event a success.
      </p>
      <div
        class="copybtnSection"
      >
        <button
          class="copybtn"
          type="submit"
        >
          <span>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-copy "
              data-icon="copy"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M208 0H332.1c12.7 0 24.9 5.1 33.9 14.1l67.9 67.9c9 9 14.1 21.2 14.1 33.9V336c0 26.5-21.5 48-48 48H208c-26.5 0-48-21.5-48-48V48c0-26.5 21.5-48 48-48zM48 128h80v64H64V448H256V416h64v48c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V176c0-26.5 21.5-48 48-48z"
                fill="currentColor"
              />
            </svg>
              Copy
          </span>
        </button>
      </div>
      <p />
    </div>
  </div>
</DocumentFragment>
`;

exports[`renders InfoSection component correctly for /contact route 1`] = `
<DocumentFragment>
  <div
    class="info-section"
  >
    <div
      class="infoNav-section"
    >
      <div
        class="back-button"
      >
        <a
          aria-current="page"
          class="links infonav-icons active"
          href="/"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-house "
            data-icon="house"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 576 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M575.8 255.5c0 18-15 32.1-32 32.1h-32l.7 160.2c0 2.7-.2 5.4-.5 8.1V472c0 22.1-17.9 40-40 40H456c-1.1 0-2.2 0-3.3-.1c-1.4 .1-2.8 .1-4.2 .1H416 392c-22.1 0-40-17.9-40-40V448 384c0-17.7-14.3-32-32-32H256c-17.7 0-32 14.3-32 32v64 24c0 22.1-17.9 40-40 40H160 128.1c-1.5 0-3-.1-4.5-.2c-1.2 .1-2.4 .2-3.6 .2H104c-22.1 0-40-17.9-40-40V360c0-.9 0-1.9 .1-2.8V287.6H32c-18 0-32-14-32-32.1c0-9 3-17 10-24L266.4 8c7-7 15-8 22-8s15 2 21 7L564.8 231.5c8 7 12 15 11 24z"
              fill="currentColor"
            />
          </svg>
        </a>
      </div>
      <div
        class="back-button"
      >
        <div>
          <button
            aria-label="nav-open"
            class="infonav-btn opendropdown-icon"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-bars infonav-icons"
              data-icon="bars"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 96C0 78.3 14.3 64 32 64H416c17.7 0 32 14.3 32 32s-14.3 32-32 32H32C14.3 128 0 113.7 0 96zM0 256c0-17.7 14.3-32 32-32H416c17.7 0 32 14.3 32 32s-14.3 32-32 32H32c-17.7 0-32-14.3-32-32zM448 416c0 17.7-14.3 32-32 32H32c-17.7 0-32-14.3-32-32s14.3-32 32-32H416c17.7 0 32 14.3 32 32z"
                fill="currentColor"
              />
            </svg>
          </button>
          <button
            aria-label="nav-close"
            class="infonav-btn closedropdown-icon hide"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-xmark infonav-icons"
              data-icon="xmark"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 384 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M342.6 150.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L192 210.7 86.6 105.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L146.7 256 41.4 361.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L192 301.3 297.4 406.6c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L237.3 256 342.6 150.6z"
                fill="currentColor"
              />
            </svg>
          </button>
        </div>
        <div
          class="dropdown-section hide"
        >
          <a
            class="infonav-links links"
            href="/about"
          >
            About
          </a>
          <a
            class="infonav-links links"
            href="/contact"
          >
            Contact
          </a>
          <a
            class="infonav-links links"
            href="/features"
          >
            Features
          </a>
          <a
            class="infonav-links links"
            href="/privacy-policy"
          >
            Privacy Policy
          </a>
          <a
            class="infonav-links links"
            href="/faqs"
          >
            FAQs | Help
          </a>
        </div>
      </div>
    </div>
    <h2
      class="content-title"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-envelope "
        data-icon="envelope"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M48 64C21.5 64 0 85.5 0 112c0 15.1 7.1 29.3 19.2 38.4L236.8 313.6c11.4 8.5 27 8.5 38.4 0L492.8 150.4c12.1-9.1 19.2-23.3 19.2-38.4c0-26.5-21.5-48-48-48H48zM0 176V384c0 35.3 28.7 64 64 64H448c35.3 0 64-28.7 64-64V176L294.4 339.2c-22.8 17.1-54 17.1-76.8 0L0 176z"
          fill="currentColor"
        />
      </svg>
       Contact
    </h2>
    <div
      class="section"
    >
      <h3
        class="section-title"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-envelope "
          data-icon="envelope"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M48 64C21.5 64 0 85.5 0 112c0 15.1 7.1 29.3 19.2 38.4L236.8 313.6c11.4 8.5 27 8.5 38.4 0L492.8 150.4c12.1-9.1 19.2-23.3 19.2-38.4c0-26.5-21.5-48-48-48H48zM0 176V384c0 35.3 28.7 64 64 64H448c35.3 0 64-28.7 64-64V176L294.4 339.2c-22.8 17.1-54 17.1-76.8 0L0 176z"
            fill="currentColor"
          />
        </svg>
         Reach Developer
      </h3>
      <p
        class="section-content"
      >
        eloghene-otiede@geekelo.com
      </p>
      <div
        class="copybtnSection"
      >
        <button
          class="copybtn"
          type="submit"
        >
          <span>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-copy "
              data-icon="copy"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M208 0H332.1c12.7 0 24.9 5.1 33.9 14.1l67.9 67.9c9 9 14.1 21.2 14.1 33.9V336c0 26.5-21.5 48-48 48H208c-26.5 0-48-21.5-48-48V48c0-26.5 21.5-48 48-48zM48 128h80v64H64V448H256V416h64v48c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V176c0-26.5 21.5-48 48-48z"
                fill="currentColor"
              />
            </svg>
              Copy
          </span>
        </button>
      </div>
      <p />
    </div>
    <div
      class="section"
    >
      <h3
        class="section-title"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-envelope "
          data-icon="envelope"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M48 64C21.5 64 0 85.5 0 112c0 15.1 7.1 29.3 19.2 38.4L236.8 313.6c11.4 8.5 27 8.5 38.4 0L492.8 150.4c12.1-9.1 19.2-23.3 19.2-38.4c0-26.5-21.5-48-48-48H48zM0 176V384c0 35.3 28.7 64 64 64H448c35.3 0 64-28.7 64-64V176L294.4 339.2c-22.8 17.1-54 17.1-76.8 0L0 176z"
            fill="currentColor"
          />
        </svg>
         Request Features
      </h3>
      <p
        class="section-content"
      >
        passlink@geekelo.com
      </p>
      <div
        class="copybtnSection"
      >
        <button
          class="copybtn"
          type="submit"
        >
          <span>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-copy "
              data-icon="copy"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M208 0H332.1c12.7 0 24.9 5.1 33.9 14.1l67.9 67.9c9 9 14.1 21.2 14.1 33.9V336c0 26.5-21.5 48-48 48H208c-26.5 0-48-21.5-48-48V48c0-26.5 21.5-48 48-48zM48 128h80v64H64V448H256V416h64v48c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V176c0-26.5 21.5-48 48-48z"
                fill="currentColor"
              />
            </svg>
              Copy
          </span>
        </button>
      </div>
      <p />
    </div>
    <div
      class="section"
    >
      <h3
        class="section-title"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-envelope "
          data-icon="envelope"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M48 64C21.5 64 0 85.5 0 112c0 15.1 7.1 29.3 19.2 38.4L236.8 313.6c11.4 8.5 27 8.5 38.4 0L492.8 150.4c12.1-9.1 19.2-23.3 19.2-38.4c0-26.5-21.5-48-48-48H48zM0 176V384c0 35.3 28.7 64 64 64H448c35.3 0 64-28.7 64-64V176L294.4 339.2c-22.8 17.1-54 17.1-76.8 0L0 176z"
            fill="currentColor"
          />
        </svg>
         Request Developer services
      </h3>
      <p
        class="section-content"
      >
        efuelight12@gmail.com
      </p>
      <div
        class="copybtnSection"
      >
        <button
          class="copybtn"
          type="submit"
        >
          <span>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-copy "
              data-icon="copy"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M208 0H332.1c12.7 0 24.9 5.1 33.9 14.1l67.9 67.9c9 9 14.1 21.2 14.1 33.9V336c0 26.5-21.5 48-48 48H208c-26.5 0-48-21.5-48-48V48c0-26.5 21.5-48 48-48zM48 128h80v64H64V448H256V416h64v48c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V176c0-26.5 21.5-48 48-48z"
                fill="currentColor"
              />
            </svg>
              Copy
          </span>
        </button>
      </div>
      <p />
    </div>
    <div
      class="section"
    >
      <h3
        class="section-title"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-envelope "
          data-icon="envelope"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M48 64C21.5 64 0 85.5 0 112c0 15.1 7.1 29.3 19.2 38.4L236.8 313.6c11.4 8.5 27 8.5 38.4 0L492.8 150.4c12.1-9.1 19.2-23.3 19.2-38.4c0-26.5-21.5-48-48-48H48zM0 176V384c0 35.3 28.7 64 64 64H448c35.3 0 64-28.7 64-64V176L294.4 339.2c-22.8 17.1-54 17.1-76.8 0L0 176z"
            fill="currentColor"
          />
        </svg>
         Give Feedback
      </h3>
      <p
        class="section-content"
      >
        passlink@geekelo.com
      </p>
      <div
        class="copybtnSection"
      >
        <button
          class="copybtn"
          type="submit"
        >
          <span>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-copy "
              data-icon="copy"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M208 0H332.1c12.7 0 24.9 5.1 33.9 14.1l67.9 67.9c9 9 14.1 21.2 14.1 33.9V336c0 26.5-21.5 48-48 48H208c-26.5 0-48-21.5-48-48V48c0-26.5 21.5-48 48-48zM48 128h80v64H64V448H256V416h64v48c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V176c0-26.5 21.5-48 48-48z"
                fill="currentColor"
              />
            </svg>
              Copy
          </span>
        </button>
      </div>
      <p />
    </div>
  </div>
</DocumentFragment>
`;
